roles/alerta/defaults/main.yml

```diff
@@ -0,0 +1,37 @@
+---
+alerta_secret: ''
+alerta_token: ''
+alerta_server_dir: '/opt/alerta'
+alerta_wsgi_root_dir: '/var/www'
+alerta_dashboard_dir: '/var/www/alerta'
+alerta_dashboard_repo: https://github.com/alerta/angular-alerta-webui.git
+alerta_ui_release: 'v5.0.0'
+alerta_amqp_release: 'cb04fb6'
+alerta_mailer_release: 'e52f4f9'
+www_url: 'localhost'
+www_alias: 'localhost'
+alerta_main_mail: ''
+alerta_mail_from: ''
+alerta_smtp_host: ''
+alerta_smtp_port: 587
+alerta_mail_debug: true
+
+# default: use pip to install argo-alert from github repo using devel branch
+argo_alert_repo: 'git+https://github.com/ARGOeu/argo-alert@devel#egg=argoalert'
+
+argo_alert_kafka: 'kafka.node.foo'
+argo_alert_token: 'secret_token'
+
+ca_bundle: true
+hostcert: /etc/grid-security/hostcert.pem
```

Comment:
these are the firewall vars,
# firewall
cert_dir: /etc/grid-security/
cert_path: /etc/grid-security/hostcert.pem
cert_key_path: /etc/grid-security/hostkey.pem
ca_path: /etc/pki/tls/certs/
ca_path_bundle: /etc/pki/tls/certs/ca-bundle.crt

It is better to keep the same vars along all roles. Please update.